third_party/rust/repos.bzl

```diff
@@ -111,7 +111,7 @@ def rust_repos(rules_rust = None, safe_ftdi = None, serde_annotate = None):
     http_archive_or_local(
         name = "lowrisc_serde_annotate",
         local = serde_annotate,
-        sha256 = "ea8bbf4f5cedf7024d7f873798299cf6bf14f98bf8cb50eb00893f68f7f681bb",
-        strip_prefix = "serde-annotate-0.0.8",
-        url = "https://github.com/lowRISC/serde-annotate/archive/refs/tags/v0.0.8.tar.gz",
+        sha256 = "28b8d73eac07c52bacb98b3b9e2cd70470b161a76c37500a24f13bbe0e7d8705",
+        strip_prefix = "serde-annotate-32aa332f8af94bb2831cbc0ebb0e325d8bd3ded4",
+        url = "https://github.com/lowRISC/serde-annotate/archive/32aa332f8af94bb2831cbc0ebb0e325d8bd3ded4.tar.gz",
```

Comment:
Would you rather cut a `0.9.0` release of serde-annotate?

Reply:
I don't see much value provided by cutting a release than pinning to a commit SHA. We are doing similar things for many other dependencies already.

Reply:
Okay, sure, I suppose the version only matters if it's published to crates.io, thanks

Reply:
I'm not sure what @cfrantz thinks, but having that crate on crates.io might be helpful to avoid listing its dependencies in our own Cargo.toml.

Reply:
The value in using a release is that a tag or release name is human-readable and the tag or release info will contain comments explaining what the purpose of the release is.  A git commit hash isn't really human readable and leaves any person investigating history stuck with reading the commit log try to understand why a particular git-hash was selected.

Please tag a release to update serde-annotate.

I don't want to publish serde-annotate on crates.io because it is, IMHO, not ready to be published in a crate index:
- This create depends on the nightly `min_specialization` feature.  I would like to rewrite the implementation to not require  nightly.
- The `Document` enum needs some work (for parsed documents, I want to keep the offset in the input where each node was parsed so that better errors can be reported).
- The conversion to JSON or YAML should be re-implemented as a trait ([This](https://github.com/lowRISC/serde-annotate/blob/main/examples/transcode.rs#L33) type of code shows up whenever converting the desired output format).

Reply:
@cfrantz In that case, could you cut a release? I don't have permission to do so in that repository.